modules/solid_mechanics/test/tests/rate_independent_cyclic_hardening/tests

```diff
@@ -0,0 +1,90 @@
+                  'under prescribed nonsymmetric strain response response under cyclic loading within the'
+                  'finite incremental strain formulation'
+  []
+  [linear_kinharden_symmetric_strain_controlled]
+    type = csv
+    input = 'linear_kinharden_symmetric_strain_controlled.i'
+    exodiff = 'linear_kinharden_symmetric_strain_controlled_out.csv'
+    compiler = 'CLANG GCC'
+    design = 'CombinedPlasticityStressUpdate.md'
+    issues = '#28688'
+    requirement = 'The system shall compute the J2 linear kinematic plasticity stress and plastic strain '
+                  'under prescribed symmetric strain response response under cyclic loading within the'
+                  'finite incremental strain formulation'
+  []
+  [linear_kinharden_symmetric_strain_controlled]
+    type = csv
+    input = 'nonlin_isoharden_symmetric_strain_controlled.i'
+    exodiff = 'nonlin_isoharden_symmetric_strain_controlled_out.csv'
+    compiler = 'CLANG GCC'
+    design = 'CombinedPlasticityStressUpdate.md'
+    issues = '#28688'
+    requirement = 'The system shall compute the J2 nonlinear isotropic plasticity stress and plastic strain '
+                  'under prescribed symmetric strain response response under cyclic loading within the'
+                  'finite incremental strain formulation'
+  []
+  [nonlin_isokinharden_symmetric_strain_controlled]
+    type = csv
+    input = 'nonlin_isokinharden_symmetric_strain_controlled.i'
+    exodiff = 'nonlin_isokinharden_symmetric_strain_controlled_out.csv'
+    compiler = 'CLANG GCC'
+    design = 'CombinedPlasticityStressUpdate.md'
+    issues = '#28688'
+    requirement = 'The system shall compute the J2 nonlinear combined plasticity stress and plastic strain '
+                  'under prescribed symmetric strain response response under cyclic loading within the'
+                  'finite incremental strain formulation'
+  []
+  [nonlin_kinharden_nonsymmetric_strain_controlled]
+    type = csv
+    input = 'nonlin_kinharden_nonsymmetric_strain_controlled.i'
+    exodiff = 'nonlin_kinharden_nonsymmetric_strain_controlled_out.csv'
+    compiler = 'CLANG GCC'
+    design = 'CombinedPlasticityStressUpdate.md'
+    issues = '#28688'
+    requirement = 'The system shall compute the J2 nonlinear kinematic plasticity stress and plastic strain '
+                  'under prescribed nonsymmetric strain response response under cyclic loading within the'
+                  'finite incremental strain formulation'
+  []
+  [nonlin_kinharden_symmetric_strain_controlled]
+    type = csv
+    input = 'nonlin_kinharden_symmetric_strain_controlled.i'
+    exodiff = 'nonlin_kinharden_symmetric_strain_controlled_out.csv'
+    compiler = 'CLANG GCC'
+    design = 'CombinedPlasticityStressUpdate.md'
+    issues = '#28688'
+    requirement = 'The system shall compute the J2 nonlinear kinematic plasticity stress and plastic strain '
```

Comment:
The requirement needs to be specific to why this test is here.  Take a look at some other test files for examples, like modules/solid_mechanics/test/tests/ad_anisotropic_creep/tests

Reply:
Do you mean I should cite the book and the figure numbers from the book that I am referring to to validate my model through these regression tests?

Reply:
Your requirements are good.  Maybe I didn't read them carefully enough yesterday.